configs/red-inventory.yaml

```diff
@@ -0,0 +1,264 @@
+
+    # One United Database - Ubuntu 20.04
+    oneunited_dbs:
+      hosts:
+        b1_oneunited_db:
+          ansible_host: 10.1.7.4
+        b2_oneunited_db:
+          ansible_host: 10.2.7.4
+
+    # Two Blue AD - Windows Server 2003 Enterprise
+    twoblue_ads:
+      hosts:
+        b1_twoblue_ad:
+          ansible_host: 10.1.7.5
+        b2_twoblue_ad:
+          ansible_host: 10.2.7.5
+
+    # Two Blue Server - Manjaro v???
+    twoblue_servers:
+      hosts:
+        b1_twoblue_server:
+          ansible_host: 10.1.7.6
+        b2_twoblue_server:
+          ansible_host: 10.2.7.6
+
+    # Three Stars IIS - Windows Server 2012 (9200)
+    threestars_iiss:
+      hosts:
+        b1_threestars_iis:
+          ansible_host: 10.1.7.7
+        b2_threestars_iis:
+          ansible_host: 10.2.7.7
+
+    # Three Stars Files - Kali Linux v???
+    threestars_filess:
+      hosts:
+        b1_threestars_files:
+          ansible_host: 10.1.7.8
+        b2_threestars_files:
+          ansible_host: 10.2.7.8
+
+# -------------------------------
+# --------- Team Groups ---------
+# -------------------------------
+
+# --- Blue Team 1
+
+    # Linux
+    b1_local_linux:
+      hosts:
+        b1_tbl_hannah:
+        b1_tbl_linux:
+
+    b1_cloud_linux:
+      hosts:
+        b1_bigmoney_mc:
+        b1_oneunited_web:
+        b1_oneunited_db:
+        b1_twoblue_server:
+        b1_threestars_files:
+
+    b1_linux:
+      children:
+        b1_local_linux:
+        b1_cloud_linux:
+
+    # Windows
+    b1_local_windows:
+      hosts:
+        b1_tbl_teller_pc:
+        b1_tbl_account_pc:
+        b1_tbl_executive_pc:
+        b1_tbl_netserv:
+        b1_tbl_ad:
+
+    b1_cloud_windows:
+      hosts:
+        b1_twoblue_ad:
+        b1_threestars_iis:
+
+    b1_windows:
+      children:
+        b1_local_windows:
+        b1_cloud_windows:
+
+# --- Blue Team 2
+
+    # Linux
+    b2_local_linux:
+      hosts:
+        b2_tbl_hannah:
+        b2_tbl_linux:
+
+    b2_cloud_linux:
+      hosts:
+        b2_bigmoney_mc:
+        b2_oneunited_web:
+        b2_oneunited_db:
+        b2_twoblue_server:
+        b2_threestars_files:
+
+    b2_linux:
+      children:
+        b2_local_linux:
+        b2_cloud_linux:
+
+    # Windows
+    b2_local_windows:
+      hosts:
+        b2_tbl_teller_pc:
+        b2_tbl_account_pc:
+        b2_tbl_executive_pc:
+        b2_tbl_netserv:
+        b2_tbl_ad:
+
+    b2_cloud_windows:
+      hosts:
+        b2_twoblue_ad:
+        b2_threestars_iis:
+
+    b2_windows:
+      children:
+        b2_local_windows:
+        b2_cloud_windows:
+
+# -------------------------------
+# -------- Global Groups --------
+# -------------------------------
+
+# --- ALl Linux
+
+    linux_locals:
+      children:
+        b1_local_linux:
+        b2_local_linux:
+
+    linux_clouds:
+      children:
+        b1_cloud_linux:
+        b2_cloud_linux:
+
+    linux:
+      children:
+        b1_linux:
+        b2_linux:
```

Comment:
This implementation needs a user. Fixed in 20f704d545f21d958bee8956aad533db9a261ad6.